attest/core/src/traits.rs

```diff
@@ -7,14 +7,21 @@ pub(crate) use alloc::format as _alloc_format;
 
 // Re-export types our macros are using
 pub(crate) use alloc::vec::Vec;
-pub(crate) use binascii::{b64decode, b64encode, bin2hex, hex2bin};
 pub(crate) use core::{
     cmp::{Ord, Ordering},
     convert::TryFrom,
     fmt::{Debug, Display, Formatter, Result as FmtResult},
     hash::{Hash, Hasher},
 };
+
+pub(crate) use base64::decode_config_slice as b64decode;
+pub(crate) use base64::encode_config_slice as b64encode;
+pub(crate) use crate::B64_CONFIG;
+
+pub(crate) use hex::encode_to_slice as bin2hex;
+pub(crate) use hex::decode_to_slice as hex2bin;
```

Comment:
Nit: While these aliases match the old impl, they're not great names. How about `hex_encode`/`hex_decode`?